GroceryStore/GroceryStore.Models/Product.cs

```diff
@@ -7,25 +7,26 @@
 
 namespace GroceryStore.Models
 {
-    public class Product
+    public abstract class Product
     {
         public string Name { get; set; }
         public string Category { get; set; }
         public double Price { get; set; }
-        public int Amount { get; set; }
+        protected DateTime ExpirationDate { get; private set; } = DateTime.Now.AddDays(1);
```

Comment:
![image](https://github.com/Shynzyn/Grocery-Store-C-Programming-Project/assets/49045181/669463db-bcd6-4333-902f-ca5cfb3c088f)
`DateTime` can't be compiled like this

Reply:
Oh I guess it can be done like this.. :D 
![image](https://github.com/Shynzyn/Grocery-Store-C-Programming-Project/assets/49045181/ceb337a5-1d31-4600-95a9-015838e99740)